terraform/aws/environments/dev/BUILD

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 Google LLC
```

Comment:
The files in this `dev` directory are pretty much copies of the files in https://github.com/privacysandbox/coordinator-services-and-shared-libraries/tree/main/operator/terraform/aws/environments/demo with some tweaks. See NEXTROLL_CHANGES.md for an account of the steps used to produce these files.